Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -491,6 +491,12 @@ Function Main {
         }
         #PROVISION VMS FOR LISA WILL ENABLE ROOT USER AND WILL MAKE ENABLE PASSWORDLESS AUTHENTICATION ACROSS ALL VMS IN SAME HOSTED SERVICE.
         Provision-VMsForLisa -allVMData $AllVMData -installPackagesOnRoleNames "none"
+        $sts = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username "root" -password $password `
+             -command "lsmod | grep hv_vmbus || echo 1" -ignoreLinuxExitCode:$true
```

Comment:
Like discussed by other folks in the thread, the lsmod method - especially for hv_vmbus - is not that good.

Instead, you would look in the kernel config directly, like from other scripts:
`vmbusIncluded=$(grep CONFIG_HYPERV=y /boot/config-$(uname -r))`

Reply:
@xian123 there is a complex situation here, the config file location is different for clearos, and the config file name like this can't apply to upstream kernel.

Reply:
Using the existed function to fix it.